scripts/qa_crowbarsetup.sh

```diff
@@ -2963,6 +2963,9 @@ function deploy_single_proposal()
                 fi
             fi
             if iscloudver 6M9plus ; then
+                # we skip all the rest of the manila setup in case of an upgrade
+                # as the images and sec-groups are already in place
+                [[ -n $upgrade_cloudsource ]] && continue
```

Comment:
> Wouldn't it be better to check if the image is already there?

that happens actually (with a wget option) in `oncontroller_manila_generic_driver_setup`.
I think here we can rely more on the fact, that this can be skipped in an upgrade (as it has been executed before)

first I had this line in `oncontroller_manila_generic_driver_setup`, but @toabctl suggested to not even go so far, which to me makes sense as well